src/Microsoft.ML.DataView/VBuffer.cs

```diff
@@ -228,50 +294,79 @@ public void CopyTo(Span<T> dst, int ivDst, T defaultValue = default(T))
+                source.AsSpan(sourceIndex, length).CopyTo(editor.Values);
             }
-            dst = editor.Commit();
+            destination = editor.Commit();
         }
 
+        /// <summary>
+        /// Returns the joint list of all index/value pairs.
+        /// </summary>
+        /// <param name="all">If <see langword="true"/> all pairs, even those implicit values of a sparse representation,
+        /// will be returned, with the implicit values having the default value, as is appropriate. If left
+        /// <see langword="false"/> then only explicitly defined values are returned.</param>
+        /// <returns>The index/value pairs.</returns>
         public IEnumerable<KeyValuePair<int, T>> Items(bool all = false)
         {
             return Items(_values, _indices, Length, _count, all);
         }
 
+        /// <summary>
+        /// Returns an enumerable with <see cref="Length"/> items, representing the values.
+        /// </summary>
         public IEnumerable<T> DenseValues()
         {
             return DenseValues(_values, _indices, Length, _count);
         }
 
-        public void GetItemOrDefault(int slot, ref T dst)
+        /// <summary>
+        /// Gets the item stored in this structure. In the case of a dense vector this is a simple lookup.
+        /// In the case of a sparse vector, it will try to find the entry with that index, and set <paramref name="destination"/>
+        /// to that stored value, or if no such value was found, assign it the default value.
+        /// </summary>
+        /// <remarks>
+        /// In the case where <see cref="IsDense"/> is <see langword="true"/>, this will take constant time since it an
+        /// directly lookup. For sparse vectors, however, because it must perform a bisection seach on the indices to
+        /// find the appropriate value, that takes logarithmic time with respect to the number of explicitly represented
+        /// items, which is to say, the <see cref="ReadOnlySpan{Int32}.Length"/> of the return value of <see cref="GetIndices"/>.
+        ///
+        /// For that reason, a single completely isolated lookup, since constructing <see cref="ReadOnlySpan{T}"/> as
+        /// <see cref="GetValues"/> does is not a free operation, it may be more efficient to use this method. However
+        /// if one is doing a more involved computation involving many operations, it may be faster to utiltize
+        /// <see cref="GetValues"/> and, if appropriate, <see cref="GetIndices"/> directly.
+        /// </remarks>
+        /// <param name="slot">The slot index, which must be a non-negative number less than <see cref="Length"/>.</param>
+        /// <param name="destination">The value stored at that index, or if this is a sparse vector where this is an implicit
+        /// entry, the default value for <typeparamref name="T"/>.</param>
+        public void GetItemOrDefault(int slot, ref T destination)
```

Comment:
Should we rename `slot`? Everywhere else we call this `index`.... We even had to specify in the param doc: `The slot index`.

Reply:
Geez, all right, I guess if we're renaming `src` and whatnot we may as well change this too while we're at it.